### PR TITLE
feat(serverless-plugin-elasticsearch-index): add plugin - EUBFR-101

### PR DESCRIPTION
**Pull request by kalinchernev**
Ready for review, there shouldn't be any breaking changes.

To test:
- run `yarn` to trigger lerna for linking new dependencies
- make deployment for a **new** stage
- verify that your stage-named index and its mapping is present at [dev-projects](https://eu-central-1.console.aws.amazon.com/es/home?region=eu-central-1#dev-projects:dashboard) domain

### CODE REVIEW

**Comment by kalinchernev** [comment]
Env vars resolution part needs to be improved.

File: services/value-store/projects/package.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
     "babel-core": "6.26.0",
     "babel-loader": "7.1.2",
     "babel-preset-env": "1.6.1",
+    "dotenv": "4.0.0",
```
Comment (yhuard) [inline services/value-store/projects/package.json]: i guess you can remove it now

File: plugins/serverless-plugin-elasticsearch-index/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Use the `custom` section in `serverless.yml` to apply settings to the plugin:
+
+```yaml
+custom:
+  slsEsIndex:
+    type: project
+    index: projects
+    mapping: ${file(./src/mappings/project.js)}
+    region: ${opt:region, file(../../../config.json):region, 'eu-central-1'}
+    domain: ${file(../elasticsearch/.serverless/stack-output.json):ServiceEndpoint, env:SLS_ES_DOMAIN}
```
Comment (yhuard) [inline plugins/serverless-plugin-elasticsearch-index/README.md]: This is not accurate anymore ;)

**Review by yhuard — COMMENTED** [review]
Some remarks before I test it

**Review by yhuard — APPROVED** [review]

**Pull request merged**